source/dub/internal/vibecompat/inet/url.d

```diff
@@ -217,6 +217,11 @@ struct URL {
 		{
 			dst.put("//");
 		}
+		if( !m_username.empty ) {
+            dst.put(m_username);
```

Comment:
This needs to use `std.uri.encodeComponent` to properly work for user names/passwords with special characters (interestingly, this is also wrong in the vibe.d implementation).

Reply:
Got it. Made the change.